apiMethods/articlesMethods.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Created by camel on 2016-01-06.
+ */
+var models = require('../models');
+
+module.exports = (function () {
+    return {
+        getSingleArticle: function (req, res) {
+            models.articles.getSingleArticle(req.params.id)
+                .then(function (article) {
+                    if (req.params.id != parseInt(req.params.id, 10)){
+                        res.send({status: 'error', error: 'Id is not an integer.'});
```

Comment:
troche w ogole bez sensu to jest, bo sprawdzasz ten parametr juz po wywolaniu zapytania do bazy 
tutaj masz cos ciekawego do walidacji parametrow https://github.com/ctavan/express-validator